vault/logical_system_activity.go

```diff
@@ -218,7 +219,7 @@ func (b *SystemBackend) handleClientExport(ctx context.Context, req *logical.Req
 		return logical.ErrorResponse("no activity log present"), nil
 	}
 
-	startTime, endTime, err := parseStartEndTimes(a, d)
+	startTime, endTime, err := parseStartEndTimes(a, d, b.Core.BillingStart())
```

Comment:
I actually talked to Vishal about this - he said that he would prefer to return start time that has the first available data (which is what it will do now).